src/index.ts

```diff
@@ -4,7 +4,7 @@ import * as iam from 'aws-cdk-lib/aws-iam';
 import * as lambda from 'aws-cdk-lib/aws-lambda';
 import * as logs from 'aws-cdk-lib/aws-logs';
 import * as rds from 'aws-cdk-lib/aws-rds';
-import * as cdk from 'aws-cdk-lib/core';
+import * as cdk from 'aws-cdk-lib/';
```

Comment:
```ts
import * as path from 'path';
import * as cdk from 'aws-cdk-lib';
import * as ec2 from 'aws-cdk-lib/aws-ec2';
import * as iam from 'aws-cdk-lib/aws-iam';
import * as lambda from 'aws-cdk-lib/aws-lambda';
import * as logs from 'aws-cdk-lib/aws-logs';
import * as rds from 'aws-cdk-lib/aws-rds';
import * as cr from 'aws-cdk-lib/custom-resources';
import { Construct } from 'constructs';
```
for the eslint~

Reply:
Thank you!
How do I test locally before committing?
I'm not very much into Projen yet..

Reply:
please run yarn build , for build and testing.

Reply:
I keep getting build errors regarding files changed during the build, can you help me with that?

Reply:
This repo will upgrade depend packages every morning. 

Because  you modify the name of stack in testing, you need also update expected template too.